Libraries/LibWeb/HTML/HTMLScriptElement.cpp

```diff
@@ -487,6 +488,7 @@ void HTMLScriptElement::prepare_script()
         // -> "importmap"
         else if (m_script_type == ScriptType::ImportMap) {
             // 1. Let result be the result of creating an import map parse result given source text and base URL.
+            prepare_to_run_script(realm());
```

Comment:
Elsewhere, `HTML::TemporaryExecutionContext` is used when a similar pattern to this is needed.

Does adding `HTML::TemporaryExecutionContext execution_context { global.realm() };` before the `exception_to_throw_completion()` call work?